tracing/providers/jaeger/src/main/java/io/helidon/tracing/providers/jaeger/JaegerTracerBuilder.java

```diff
@@ -335,13 +351,61 @@ public JaegerTracerBuilder trustedCertificates(io.helidon.common.configurable.Re
         return this;
     }
 
+    /**
+     * Span Processor type used.
+     *
+     * @param spanProcessorType to use
+     * @return updated builder
+     */
+    @ConfiguredOption(key = "span-processor-type", value = "batch")
```

Comment:
do not define an explicit key if it matches the method name. The one you defined will be automatically used (a captical case letter is replaced with `-` and lower cased letter)

Reply:
ok